spec/forms/prior_authority/steps/primary_quote_form_spec.rb

```diff
@@ -0,0 +1,121 @@
+require 'rails_helper'
+
+RSpec.describe PriorAuthority::Steps::PrimaryQuoteForm do
+  subject(:form) { described_class.new(arguments) }
+
+  let(:arguments) do
+    {
+      record:,
+      service_name:,
+      contact_full_name:,
+      organisation:,
+      postcode:,
+    }
+  end
+
+  describe '#validate' do
+    let(:record) { instance_double(Quote) }
+
+    context 'with valid quote details' do
+      let(:service_name) { 'Forensics Expert' }
+      let(:contact_full_name) { 'Joe Bloggs' }
+      let(:organisation) { 'LAA' }
+      let(:postcode) { 'CR0 1RE' }
+
+      it { is_expected.to be_valid }
+    end
+
+    context 'with blank quote details' do
+      let(:service_name) { '' }
+      let(:contact_full_name) { '' }
+      let(:organisation) { '' }
+      let(:postcode) { '' }
+
+      it 'has a validation errors on blank fields' do
+        expect(form).not_to be_valid
+        expect(form.errors.of_kind?(:service_name, :blank)).to be(true)
+        expect(form.errors.of_kind?(:contact_full_name, :blank)).to be(true)
+        expect(form.errors.of_kind?(:organisation, :blank)).to be(true)
+        expect(form.errors.of_kind?(:postcode, :blank)).to be(true)
+        expect(form.errors.messages.values.flatten)
+          .to include('Enter the service required',
+                      "Enter the contact's full name",
+                      'Enter the organisation name',
+                      'Enter the postcode')
+      end
+    end
+
+    context 'with invalid quote details' do
+      let(:service_name) { 'Forensics Expert' }
+      let(:contact_full_name) { 'Tim' }
+      let(:organisation) { 'LAA' }
+      let(:postcode) { 'loren ipsum' }
+
+      it 'has a validation errors on blank fields' do
+        expect(form).not_to be_valid
+        expect(form.errors.of_kind?(:contact_full_name, :invalid)).to be(true)
+        expect(form.errors.of_kind?(:postcode, :invalid)).to be(true)
+        expect(form.errors.messages.values.flatten)
+          .to include('Enter a valid full name',
+                      'Enter a real postcode')
+      end
+    end
+  end
+
+  describe '#save' do
+    subject(:save) { form.save }
+
+    let(:record) { create(:quote, :blank, prior_authority_application: create(:prior_authority_application)) }
+
+    context 'with valid quote details' do
+      let(:service_name) { 'Forensics Expert' }
+      let(:contact_full_name) { 'Joe Bloggs' }
+      let(:organisation) { 'LAA' }
+      let(:postcode) { 'CR0 1RE' }
+
+      # rubocop:disable RSpec/ExampleLength
```

Comment:
To me, this is a sign instead of disabling the cop inline we should be configuring rubocop to allow examples of this length. If we think this is a reasonable length for an example (which I do), then it's pointless to have a rule telling us off for example of this length that we have to keep ignoring. If we don't think this is a reasonable length for an example, we should change this example to make it more concise.